README.md

```diff
@@ -26,35 +26,35 @@
 
- 1. It brings potential risks to break the website if there is any compatible issue or bug coming from upstream. It's not a good idea to use it in the production environment, both jQuery CDN and Google CDN don't provide this feature.
- 2. The `latest` url feature also has cache and performance issues, which has conflicts of what we are trying to do - make your website load as fast as possible.
- 3. Security issues become more and more important nowadays, for example, GitHub had been attacked by the Great Cannon via malicious JavaScript([ref1](https://citizenlab.org/2015/04/chinas-great-cannon/), [ref2](https://arstechnica.com/security/2015/04/meet-great-cannon-the-man-in-the-middle-weapon-china-used-on-github/)) during April 2015, one of the techniques to prevent this kind of attack is [Subresource Integrity(SRI)](https://developer.mozilla.org/en-US/docs/Web/Security/Subresource_Integrity). We have been supporting this feature for a long time, but the `latest` url feature totally conflicts with the SRI technique.
+ 1. Your website might break if there are any compatibility issues or bugs coming from upstream. It's not a good idea to use it in the production environment. Both jQuery CDN and Google CDN don't provide this feature.
+ 2. The `latest` URL feature also has cache and performance issues. This conflicts with what we are trying to do - make your website load as fast as possible.
+ 3. Security issues have become more and more important nowadays. For example, Great Cannon attacked GitHub via malicious JavaScript ([ref1](https://citizenlab.org/2015/04/chinas-great-cannon/), [ref2](https://arstechnica.com/security/2015/04/meet-great-cannon-the-man-in-the-middle-weapon-china-used-on-github/)) during April 2015. One of the techniques to prevent this kind of attack is [Subresource Integrity (SRI)](https://developer.mozilla.org/en-US/docs/Web/Security/Subresource_Integrity). We have been supporting this feature for a long time. The `latest` URL feature, however, totally conflicts with the SRI technique.
 
 ## Contributing
 
-To contribute to CDNJS, please refer to the [CONTRIBUTING.md](https://github.com/cdnjs/cdnjs/blob/master/CONTRIBUTING.md), it should contain most of the things you'll need to get your contribution started!
+To contribute to CDNJS, please refer to [CONTRIBUTING.md](https://github.com/cdnjs/cdnjs/blob/master/CONTRIBUTING.md). It should contain most of the things you'll need to get your contribution started!
 
-**Working on your first Pull Request?** You can learn how from this *free* series [How to Contribute to an Open Source Project on GitHub](https://egghead.io/series/how-to-contribute-to-an-open-source-project-on-github)
+**Working on your first Pull Request?** Learn how from this *free* series: [How to Contribute to an Open Source Project on GitHub](https://egghead.io/series/how-to-contribute-to-an-open-source-project-on-github)
```

Comment:
This was copied from http://makeapullrequest.com/ actually, but I won't mind if we have better wording.